pkg/resources/share.go

```diff
@@ -28,10 +28,11 @@ var shareSchema = map[string]*schema.Schema{
 		Description: "Specifies a comment for the managed account.",
 	},
 	"accounts": &schema.Schema{
-		Type:        schema.TypeSet,
-		Elem:        &schema.Schema{Type: schema.TypeString},
-		Optional:    true,
-		Description: "A list of accounts to be added to the share.",
+		Type:             schema.TypeList,
```

Comment:
can you add a comment linking that github issue you found last night?

Reply:
ah ok I thought you meant in the PR

Reply:
I feel like this makes it a breaking change, can you prefix your pull request title with `[breaking]`